lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -443,6 +443,16 @@ span.ltx_framed       { display:inline-block; text-indent:0; } /* avoid padding/
 .ltx_svg_fog foreignObject > p { margin:0; padding:0; display:block; }
 /*.ltx_svg_fog foreignObject > p { margin:0; padding:0; display:block; white-space:nowrap; }*/
 
+.ltx_foreignobject_container { display:flex; width:100%; height:100%; align-items: end; justify-content: center; justify-items:center; }
+@supports not (text-box: trim-both cap alphabetic) {
+  .ltx_foreignobject_content { line-height: calc(var(--lx-foreignobject-height) - var(--lx-foreignobject-depth)); translate: 0 var(--lx-foreignobject-depth); }
+  .ltx_foreignobject_content .ltx_inline-block { line-height: initial; } }
+@supports (text-box: trim-both cap alphabetic) {
+  .ltx_foreignobject_content { text-box: trim-both cap alphabetic; } }
```

Comment:
I have reduced Safari's rendering issue with multiline content to some weird interaction between `text-box` and `vertical-align`:
https://codepen.io/xworld21/pen/mydzXjo
I suspect this to be a WebKit bug.

(Context: I have added a two-line `\vbox{}` inside the foreign object. Chrome works as expected but Safari acts in a very weird way.)

Reply:
Yes, it was a WebKit bug, and it has been fixed incredibly quickly. Not sure how long before it ships.
https://bugs.webkit.org/show_bug.cgi?id=290727